Origin/feature/pagination

Romano1994o opened the pull request:
додав pattern  до модального вікна в header.html

Romano1994o committed:
пробував поміняти border в пагінації завтра дороблю

Co-Authored-By: Dmytro Ivashchenko <37770603+biggujo@users.noreply.github.com>

Romano1994o committed:
1)допобив стилі для кнопок

Co-Authored-By: Dmytro Ivashchenko <37770603+biggujo@users.noreply.github.com>

Romano1994o committed:
поправив пагінацію з border na стрілках і додав борлек при ховері на стрілки

Romano1994o committed:
додав стилі до кнопок

biggujo commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Валідація Order Now](https://trello.com/c/dAR8ER2A/92-%D0%B2%D0%B0%D0%BB%D1%96%D0%B4%D0%B0%D1%86%D1%96%D1%8F-order-now)

biggujo commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Пагінація](https://trello.com/c/qWhTIlHU/56-%D0%BF%D0%B0%D0%B3%D1%96%D0%BD%D0%B0%D1%86%D1%96%D1%8F)

biggujo reviewed (changes requested):
It is unclear how to use pagination. Validation is ok

Romano1994o committed:
button.css

доробив стилі на кнопках щоб правильно відображались на різних розмірах екрану

Co-Authored-By: Dmytro Ivashchenko <37770603+biggujo@users.noreply.github.com>

Romano1994o committed:
scrollbar

зробив класи для скрол бару горизонтального і як приклад додав кнопки в favorite.html щоб наглядно можна було чекнути

Co-Authored-By: Dmytro Ivashchenko <37770603+biggujo@users.noreply.github.com>

Romano1994o committed:
Update favorites.html

для перевірки горизонтального csroll

Co-Authored-By: Dmytro Ivashchenko <37770603+biggujo@users.noreply.github.com>

Romano1994o committed:
Update index.html

видалив зайве підключення all-categories

Co-Authored-By: Dmytro Ivashchenko <37770603+biggujo@users.noreply.github.com>

Romano1994o committed:
fix

Co-Authored-By: Dmytro Ivashchenko <37770603+biggujo@users.noreply.github.com>

biggujo committed:
Remove overflow: hidden on body

biggujo commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Скрол-бар](https://trello.com/c/haUkgzHh/60-%D1%81%D0%BA%D1%80%D0%BE%D0%BB-%D0%B1%D0%B0%D1%80)

biggujo commented:
![](https://github.trello.services/images/mini-trello-icon.png) [UI kit part 2 (потрібні лише кнопки та пагінація)](https://trello.com/c/IP6doGr5/23-ui-kit-part-2-%D0%BF%D0%BE%D1%82%D1%80%D1%96%D0%B1%D0%BD%D1%96-%D0%BB%D0%B8%D1%88%D0%B5-%D0%BA%D0%BD%D0%BE%D0%BF%D0%BA%D0%B8-%D1%82%D0%B0-%D0%BF%D0%B0%D0%B3%D1%96%D0%BD%D0%B0%D1%86%D1%96%D1%8F)

biggujo committed:
Merge branch 'main' into origin/feature/pagination
